locksmith/src/operations/stripeOperations.ts

```diff
@@ -59,7 +59,35 @@ export const saveStripeCustomerIdForAddress = async (
   }
 }
 
+/**
+ * Method which delets the stripe customer id for an address
+ */
+export const deletePaymentDetailsForAddress = async (
+  publicKey: ethereumAddress
+) => {
+  const normalizedEthereumAddress = Normalizer.ethereumAddress(publicKey)
+
+  // First, let's delete the StripeCustomer
+  const deletedStripeCustomer = await StripeCustomer.destroy({
```

Comment:
Not sure of the exact impetus for this feature but, for the sake of completeness the card should probably be removed from stripe, I do believe that their console allows for charges to be created there.

Reply:
This is something that I was thinking we should do at another level.

Reply:
I'll ad this code.

Reply:
It requires a bit more work since we currently do not keep track of the `source` specificially, so that means we would need to list all sources and remove them all.